packages/mephisto-worker-experience/README.md

```diff
@@ -0,0 +1,70 @@
+# mephisto-worker-experience
+
+This package provides the Tips component to aid in the creation of user-interfaces for Mephisto
```

Comment:
"user-interfaces for Mephisto" is a bit vague. Feel free to borrow verbiage from the Project Plan or the tasks tool.

> The Tips component will allow task authors to set up a communication channel to solicit "tips" from workers to share with other workers, thus allowing for the "crowdsourcing" of the instructions for tasks as well. We find that workers sometimes will share these tips in third-party forums or via emails to the task authors. This feature creates a more vetted channel for such communication.